.github/workflows/test.yml

```
@@ -0,0 +1,135 @@
+    - name: Run E2E tests
+      run: npm run test:e2e
+      
+    - name: Run type checking
+      run: npm run lint
+      
+    - name: Generate coverage report
+      run: npm run test:coverage
+      if: matrix.node-version == '20.x' && matrix.os == 'ubuntu-latest'
+      
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)